editor/src/components/inspector/sections/style-section/style-section.tsx

```diff
@@ -8,6 +8,14 @@ import { ShadowSubsection } from './shadow-subsection/shadow-subsection'
 import { TextShadowSubsection } from './text-subsection/text-shadow-subsection'
 import { AutosizingTextSubsection } from './text-subsection/text-subsection'
 import { TransformSubsection } from './transform-subsection/transform-subsection'
+import {
+  FlexRow,
+  InspectorSectionHeader,
+  InspectorSubsectionHeader,
+  Section,
+} from '../../../../uuiui'
+import { GridRow } from '../../widgets/grid-row'
+import { InspectorContextMenuWrapper } from '../../../../uuiui-deps'
```

Comment:
it seems like all these new imports were not used in the end, please remove them